migrations/101_distrubute_ether_to_fixtures_accounts.js

```diff
@@ -0,0 +1,29 @@
+require("babel-register");
+const getConfig = require("./config").getConfig;
+const getFixtureAccounts = require("./config").getFixtureAccounts;
+const getDeployerAccount = require("./config").getDeployerAccount;
+const promisify = require("../test/helpers/evmCommands").promisify;
+
+module.exports = function deployContracts(deployer, network, accounts) {
```

Comment:
you should do it before  `migrations/7_patch_icbm_state.js`. could be even in first step which is empty now